util/text/left_padded_test.go

```diff
@@ -28,13 +28,13 @@ func TestLeftPadMaxLine(t *testing.T) {
 		},
 		{
 			"foo",
-			"  f…",
+			"  ...",
```

Comment:
Please keep using the `…` character, it allows to save space and have more useful characters displayed. It's very helpful on a small terminal area.

Reply:
The problem here is that '…' is counted two cells but displayed as '…' + VOID in the terminal. This makes the background of the selected item in bug table missing one char:
<img width="939" alt="default" src="https://user-images.githubusercontent.com/32396/50569341-9b6ab900-0d9d-11e9-9996-83313d42256e.png">

I guess this can be resolved in gocui? If it can be fixed there, I am very glad to change it back to unicode ellipsis.

Reply:
The problem here is that '…' is counted two cells but displayed as '…' + VOID in the terminal. This makes the background of the selected item in bug table missing one char:
<img width="939" alt="default" src="https://user-images.githubusercontent.com/32396/50569341-9b6ab900-0d9d-11e9-9996-83313d42256e.png">

I guess this can be resolved in gocui? If it can be fixed there, I am very glad to change it back to unicode ellipsis.

Reply:
Luckily runewidth exports some variables so we can workaround this in our code.